src/samples/MQTTClient_publish_async.c

```diff
@@ -65,7 +65,10 @@ int main(int argc, char* argv[])
     MQTTClient_deliveryToken token;
     int rc;
 
-    if ((rc = MQTTClient_create(&client, ADDRESS, CLIENTID,
+	const char* uri = (argc > 1) ? argv[1] : ADDRESS;
```

Comment:
ident issue.

Reply:
Argh. Yeah, you're correct. There is inconsistent formatting in this project. Why are the samples using spaces when the rest of the code is using tabs?